src/lib/mongoBackend/MongoGlobal.cpp

```diff
@@ -2004,7 +2005,34 @@ bool entitiesQuery
   auto_ptr<DBClientCursor>  cursor;
   Query                     query(finalQuery.obj());
 
-  query.sort(BSON(ENT_CREATION_DATE << 1));
+  if (sorted == "")
```

Comment:
Up to you, your code ... :-)

I would change the name of the variable.

NTC, if you wish :-)